PopPUNK/__main__.py

```diff
@@ -101,6 +109,8 @@ def get_options():
     iGroup.add_argument('--distances', help='Prefix of input pickle of pre-calculated distances')
     iGroup.add_argument('--external-clustering', help='File with cluster definitions or other labels '
                                                       'generated with any other method.', default=None)
+    iGroup.add_argument('--viz-lineages', help='CSV with lineage definitions to use for visualisation'
```

Comment:
Better to move visualisation to another exported script?

Reply:
Agree. Recorded as code issue #77.